chrysler_pacifica_ehybrid_s_2019/docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
     command: roscore
 
   platform:
-    image: ${DOCKER_ORG}/carma-platform:${DOCKER_TAG}
+    image: local/carma-platform:latest
```

Comment:
Are there any updates to carma-platform that aren't in develop now, or that should be?

Reply:
There are some updates needed for functionality-wise.
Rest are just parameter changes.
Here is one PR for cooperative_lanechange (I havent had the chance to retest that is why it hasn't closed yet): https://github.com/usdot-fhwa-stol/carma-platform/pull/2531
Here is another PR that should stop it from shutting down during yielding
https://github.com/usdot-fhwa-stol/carma-platform/pull/2535

Reply:
All PRs are closed except the one that shouldn't be merged probably, but is needed for the workzone + CP demo: https://github.com/usdot-fhwa-stol/carma-platform/pull/2537